build.gradle

```diff
@@ -26,6 +28,25 @@ dependencies {
     compile "com.xenomachina:kotlin-argparser:2.+"
     compile "com.spotify:docker-client:8.11.+"
     compile "com.google.cloud:google-cloud-pubsub:0.32.0-beta"
+    compile 'com.google.protobuf:protobuf-java:3.5.+'
+    compile 'io.grpc:grpc-protobuf:1.+'
```

Comment:
This is likely to become needed soon, but not now. Please comment it out and add explanation: "uncomment when we start doing grpc requests"

Reply:
Done